doc/classes/TextEdit.xml

```diff
@@ -362,6 +362,24 @@
 				Returns OpenType feature [code]tag[/code].
 			</description>
 		</method>
+		<method name="get_pos_at_line_column" qualifiers="const">
```

Comment:
We renamed all `pos` to `position` in 3.0, should probably be kept consistent (both here and in argument names).

Reply:
While I agree with your point, I was just following the established scheme for other related methods :eyes:

![image](https://user-images.githubusercontent.com/11782833/142861051-8c1832e6-5e8f-499c-9826-711f30971c52.png)